views/view-aquilon-sandboxes.php

```diff
@@ -51,9 +51,9 @@
     foreach ($panel as $owner => $nodes) {
         if ($group != "domain") {
             $realname = $owner;
-            $id = $CONFIG['ID']['PATH'];
-            if (file_exists($id)) {
-                $realname = exec("$id $owner");
+            $owner_id = $CONFIG['ID']['PATH'];
```

Comment:
Nope, `$CONFIG['ID']['PATH']` is the location of the user-name lookup script, in our case this is the federal id lookup tool.
`$username_lookup_script`...?